apps/researcher/src/lib/community.ts

```diff
@@ -0,0 +1,91 @@
+import {clerkClient, auth} from '@clerk/nextjs';
+
+type Community = {
+  id: string;
+  name: string;
+};
+
+export type Membership = {
+  id: string;
+  role: string;
+  publicUserData?: {
```

Comment:
Would it be possible to use a more descriptive name instead? (`publicUserData` sounds a bit vague - what is it?) For example: `user` or `member`?